### PR TITLE
feat:add example.yaml for release

### PR DESCRIPTION
**Pull request by fredrikliu**
**Please provide issue(s) of this PR:**
add example.yaml for release,Delete unnecessary configuration information
**To help us figure out who should review this PR, please put an X in all the areas that this PR affects.**

- [ ] Docs
- [ ] Discovery
- [ ] Route
- [ ] RateLimit
- [ ] CircuitBreaker
- [x] Test and Release

**Please check any characteristics that apply to this pull request.**

- [ ] Does not have any user-facing changes. This may include API changes, behavior changes, performance improvements, etc.

### CODE REVIEW

**Comment by fredrikliu** [comment]
three things：
1. add example.yaml
2. change application.propertities,delete unnecessary information

File: polaris-agent-examples/spring-cloud-plugins-examples/spring-cloud-hoxton-examples/quickstart-examples/provider-a/src/main/resources/polaris.yml [hunk 1]
```diff
@@ -0,0 +1,8 @@
+consumer:
+  serviceRouter:
```
Comment (andrewshan) [inline polaris-agent-examples/spring-cloud-plugins-examples/spring-cloud-hoxton-examples/quickstart-examples/provider-a/src/main/resources/polaris.yml:2]: provider不需要就近级别配置

File: polaris-agent-examples/spring-cloud-plugins-examples/spring-cloud-hoxton-examples/quickstart-examples/provider-b/src/main/resources/polaris.yml [hunk 2]
```diff
@@ -0,0 +1,8 @@
+consumer:
+  serviceRouter:
```
Comment (andrewshan) [inline polaris-agent-examples/spring-cloud-plugins-examples/spring-cloud-hoxton-examples/quickstart-examples/provider-b/src/main/resources/polaris.yml:2]: provider不需要就近级别配置

**Review by andrewshan — APPROVED** [review]
LGTM

**Pull request merged**